src/frontend/src/tests/sol/services/sol-address.services.spec.ts

```diff
@@ -0,0 +1,178 @@
+import { SOLANA_KEY_ID } from '$env/networks/networks.sol.env';
+import { SOLANA_TOKEN_ID } from '$env/tokens/tokens.sol.env';
+import * as idbApi from '$lib/api/idb.api';
+import * as signerApi from '$lib/api/signer.api';
+import {
+	solAddressDevnetStore,
+	solAddressLocalnetStore,
+	solAddressMainnetStore,
+	solAddressTestnetStore
+} from '$lib/stores/address.store';
+import { authStore } from '$lib/stores/auth.store';
+import * as toastsStore from '$lib/stores/toasts.store';
+import { LoadIdbAddressError } from '$lib/types/errors';
+import { replacePlaceholders } from '$lib/utils/i18n.utils';
+import { SOLANA_DERIVATION_PATH_PREFIX } from '$sol/constants/sol.constants';
+import {
+	getSolAddressDevnet,
+	getSolAddressLocal,
+	getSolAddressMainnet,
+	getSolAddressTestnet,
+	loadIdbSolAddressMainnet,
+	loadSolAddressDevnet,
+	loadSolAddressLocal,
+	loadSolAddressMainnet,
+	loadSolAddressTestnet,
+	validateSolAddressMainnet
+} from '$sol/services/sol-address.services';
+import { SolanaNetworks } from '$sol/types/network';
+import en from '$tests/mocks/i18n.mock';
+import { Ed25519KeyIdentity } from '@dfinity/identity';
+import { getAddressDecoder } from '@solana/addresses';
+import { get } from 'svelte/store';
+import type { MockInstance } from 'vitest';
+
+vi.mock('@solana/addresses', () => ({
+	getAddressDecoder: vi.fn()
+}));
+
+describe('sol-address.services', () => {
+	const mockSolAddress = 'solana123';
+	const mockIdentity = Ed25519KeyIdentity.generate();
```

Comment:
How about using `mockIdentity` from `identity.mock.ts`?

Reply:
ah yes, sry, I did not adjust it here